### ISSUE TITLE
Is there fix like that but for MGS1?

### ISSUE THREAD

**Issue by notthefox** [VFansss/mgs2-v-s-fix#139]
i love this patch so much, but there is something similar for the first part?

**Comment by Foxiravoc** [comment]
This is a direct link to the recent reddit post from oxide-NL (MGS1 fan-made patch creator) where he explains how to apply his patch to the GOG version of MGS1: https://www.reddit.com/r/metalgearsolid/comments/izj08e/metal_gear_metal_gear_solid_metal_gear_solid_2/g6n9weq?utm_source=share&utm_medium=web2x&context=3

**Comment by Darthagnon** [comment]
For the record, here's the link to oxide-NL's original patch release thread on Reddit:

https://www.reddit.com/r/metalgearsolid/comments/4qlv1c/final_full_release_metal_gear_solid_1_integral_pc/

**Comment by Tomasak** [comment]
MGS1 GOG only needs widescreen fix. It works fine on modern systems, has high res rendering, hw mode incompatiblity fixed, AA, AF, it even works out of box with xinput controllers even it says otherwise.

Only thing you need is to apply widescreen fix. How to is here
https://www.pcgamingwiki.com/wiki/Metal_Gear_Solid:_Integral#Widescreen_resolution

oxide-NL's "patch" is just dgvoodoo to make hw rendering working on modern HW and sweetfx for SMAA and that stupid logo change on timer, which slows you down. It doesnt even fix FMVs, thus using that thing breaks something GOG fixed.
It has glide emulation from dgvoodoo extracted, when glide api is not even used in MGS. And doesnt have widescreen fix installed

**Comment by Foxiravoc** [comment]
> It doesnt even fix FMVs, thus using that thing breaks something GOG fixed.

I'm so glad GOG fixed the FMVs. I remember whenever the blank screens came in, I switched to the folder where I stored individual video files I've recorded from the PSOne version before jumping back in-game.
Those FMVs are in DDV format which is obsolete and the only program I possess and know of that can read them externally is called Masher.exe which has long been discontinued. It appears the same files are still included in the GOG version and I wonder how they managed to make them work in-game on modern systems.
